src/test/regress/expected/propagate_extension_commands.out

```diff
@@ -385,6 +385,25 @@ SELECT run_command_on_workers($$SELECT count(*) FROM pg_extension WHERE extname
  (localhost,57638,t,2)
 (2 rows)
 
+-- test if citus can escape the extension name
+CREATE EXTENSION "uuid-ossp";
```

Comment:
before merging in, please make sure that it works with master_add_node as well?

It's good that the regression test file has a schema  name which requires escaping, so we've tested that.

Reply:
Updated propagate_extension_commands.sql to test the fix with master_add_node as well